.github/workflows/rune_centos_with_docker_and_crictl.yml

```diff
@@ -114,6 +115,69 @@ jobs:
       run: docker exec $centos bash -c "cd /root/samples && ./clean.sh;
         crictl run --timeout 3s golang.yaml pod.yaml && ./show.sh"
 
+    - name: Create skeleton bundle on centos
+      if: always()
+      run: |
```

Comment:
There is no need to add `|`.  Please check the other lines and add `|` when needed.